config/core/test-configs-chromeos.yaml

```diff
@@ -93,13 +93,92 @@ test_plans:
       job_timeout: '60'
       docker_image: 'kernelci/cros-tast'
 
-  cros-tast-fixed:
+  cros-tast-kernel:
+    <<: *cros-tast
+    params:
+      <<: *cros-tast-params
+      tests: &cros-tast-kernel-tests >
+        kernel.*
+
+  cros-tast-perf:
+    <<: *cros-tast
+    params:
+      <<: *cros-tast-params
+      tests: &cros-tast-perf-tests >
+        ui.WindowCyclePerf
+
+  cros-tast-platform:
+    <<: *cros-tast
+    params:
+      <<: *cros-tast-params
+      tests: &cros-tast-platform-tests >
+        platform.CheckDiskSpace
+        platform.CheckProcesses
+        platform.CheckTracefsInstances
+        platform.ChromeMlocked
+        platform.CrosDisks
+        platform.CrosDisksArchive
+        platform.CrosDisksFilesystem
+        platform.CrosDisksFormat
+        platform.CrosDisksRename
+        platform.CrosDisksSSHFS
+        platform.CrosID
+        platform.Crossystem
+        platform.DLCService
+        platform.DLCServiceCrosDeploy
+        platform.DLCServicePreloading
+        platform.DMVerity
+        platform.DumpVPDLog
+        platform.Firewall
+        platform.HeavyMemoryUser
+        platform.HeavyMemoryUser.vm
+        platform.Histograms
+        platform.LocalPerfettoTBMTracedProbes
+        platform.Memd
+        platform.Mosys
+        platform.Mosys.ec
+        platform.Mosys.memory
+        platform.Mtpd
+        platform.TPMResponsive
+
+  cros-tast-video:
     <<: *cros-tast
-    filters: *cros-filters-fixed
     params:
+      <<: *cros-tast-params
+      tests: &cros-tast-video-tests >
+        video.ChromeStackDecoding.*
+
+  cros-tast-fixed: &cros-tast-fixed
+    <<: *cros-tast
+    filters: *cros-filters-fixed
+    params: &cros-tast-fixed-params
       <<: *cros-tast-params
       fixed_kernel: true
 
+  cros-tast-fixed-kernel:
+    <<: *cros-tast-fixed
+    params:
+      <<: *cros-tast-fixed-params
+      tests: *cros-tast-kernel-tests
+
+  cros-tast-fixed-perf:
+    <<: *cros-tast-fixed
+    params:
+      <<: *cros-tast-fixed-params
+      tests: *cros-tast-perf-tests
+
+  cros-tast-fixed-platform:
+    <<: *cros-tast
+    params:
+      <<: *cros-tast-params
+      tests: *cros-tast-platform-tests
+
+  cros-tast-fixed-video:
+    <<: *cros-tast-fixed
+    params:
+      <<: *cros-tast-fixed-params
+      tests: *cros-tast-video-tests
```

Comment:
We can leave them here to start with but I think we should stop running the `-fixed` tests as soon as we have shown that we can run the tests with the real kernels.

Later we may start adding actual tests for Chrome OS kernels by building the downstream branches with the CrOS SDK.

Reply:
Probably we can use them similar to cros-flash? 
Just once after flashing to check what does current reference kernel can run on this platform and what it can't run, and maybe store results somewhere, so it will be easier to compare with upstream.
Or assign to some test kernel that run once per month, if this results might be cleaned up.

Reply:
My worry is that it's bloating the YAML config for not much added value.  Ultimately we should stop running the `-fixed` tests altogether, they're just a hack  but are useful for bringing up new platforms initially.

Reply:
There seems to be another issue, too
Even if we change name and order of  "fixed" and "non-fixed" test cases. `tast_quemu` test plans will bring another problem.

```
Broken order in config/core/test-configs-chromeos.yaml test_plans: 'cros-tast-video' is before 'cros-tast_qemu'
```

Maybe we should also rename them to something like `cros-tast-qemu`, `cros-tast-platform-qemu`, `cros-tast-platform-qemu-fixed`?

Reply:
I created new names and removed the *-fixed test plans as they're not bringing much value at the moment